providers/PopupWidgetProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import { createContext, useContext, useMemo, useState } from "react"
+
+const PopupWidgetContext = createContext(null)
+
+const PopupWidgetProvider = ({ children }) => {
+  const [isOpenAbout, setIsOpenAbout] = useState(false)
+  const [isOpenMusic, setIsOpenMusic] = useState(false)
+  const [isOpenWeb3, setIsOpenWeb3] = useState(false)
+  const [isOpenPress, setIsOpenPress] = useState(false)
```

Comment:
this is a lot of state

you could make this simpler by making one, single `open` variable and pass different content to make this more dynamic & composable for new page content